src/test/java/net/sf/jabref/logic/util/io/FileUtilTest.java

```diff
@@ -151,4 +167,66 @@ public void uniquePathSubstrings() {
         assertEquals(uniqPath, result);
     }
 
+    @Test
+    public void testCopyFileFromEmptySourcePathToEmptyDestinationPathWithOverrideExistFile() throws Exception{
+        assertFalse(FileUtil.copyFile(emptyTestPath, emptyTestPath, true));
+    }
+
+    @Test
+    public void testCopyFileFromEmptySourcePathToEmptyDestinationPathWithoutOverrideExistFile() throws Exception{
+        assertFalse(FileUtil.copyFile(emptyTestPath, emptyTestPath, false));
+    }
+
+    @Test
+    public void testCopyFileFromEmptySourcePathToExistDestinationPathWithOverrideExistFile() throws Exception{
+        assertFalse(FileUtil.copyFile(emptyTestPath, emptyTestFilePath, true));
+    }
+
+    @Test
+    public void testCopyFileFromEmptySourcePathToExistDestinationPathWithoutOverrideExistFile() throws Exception{
+        assertFalse(FileUtil.copyFile(emptyTestPath, emptyTestFilePath, false));
+    }
+
+    @Test
+    public void testCopyFileFromExistSourcePathToExistDestinationPathWithOverrideExistFile() throws Exception{
+
+        assertTrue(FileUtil.copyFile(emptyTestFilePath, emptyTestFilePath, true));
+    }
+
+    @Test
+    public void testCopyFileFromExistSourcePathToExistDestinationPathWithoutOverrideExistFile() throws Exception{
+        assertFalse(FileUtil.copyFile(emptyTestFilePath, emptyTestFilePath, false));
+    }
+
+    @Test (expected = NullPointerException.class)
+    public void testRenameFiletoNull() throws Exception {
```

Comment:
If you want to cover all three cases you have to put them in three different methods since any of these statements throws a NPE. This means only the first one will be executed and the test will pass without executing the remaining two.